### ISSUE TITLE
White Box Compatibility

### ISSUE THREAD

**Issue by Linsolv** [vttred/ose#166]
I was hoping to run a game of White Box FMAG and discovered there really _wasn't_ a _good_ way to do it in Foundry. I thought that probably the best and easiest way to do it would be to use OSE and then modify in the modifier tables, which seems in my limited testing to have worked somewhat. I added a little Boolean in the config menu, and just put in an if statement to check that function in the `computeModifiers()` function. I wanted to know if that was something that we could get officially added; I'd gladly send over the code I wrote, but I do apologize in advance if I wrote some janky stuff; it's been ages since I did any coding and I really just thought I'd try to slap a band-aid on. So far as I can tell it works, though there are things I couldn't figure out how to test.

Also, I didn't localize the setting text and hint; I added hooks to do it, but since I don't speak Brazilian Portuguese, French, Spanish, German, or Italian, I just copied and pasted in the English text for those localizations to get the module to run.

**Comment by anthonyronda** [comment]
The best approach to this for now would involve hard-forking the system and creating a new repository for White Hack.

It may seem like there's just one little change to make, and that one change might be fine for your own personal use, but there's probably a lot of opportunities for changes that serve White Hack players. And the maintainer of that system could be you!

I've only played Macchiato Monsters myself, a White Hack/Black Hack mashup hack.

**Comment by Linsolv** [comment]
Not Whitehack, which is quite different—but White _Box_. That is, Swords & Wizardry WhiteBox... though I personally only checked against the _White Box: Fantastic Medieval Adventure Game_ book by Charlie Mason, which is a reformatting and rewriting of, but not (so far as I can tell) an actual rules change from, S&W:WB.

That being said, I don't have any problem with your suggestion, either. Indeed, my original thought was just to cut out the OSE parts that I didn't want or need, but I thought that seemed a little rude, to take someone else's work and then do 20 minutes of work hacking a tiny thing, and slap my name next to theirs like I did something significant!

**Comment by anthonyronda** [comment]
> White *Box*

Failed reading comprehension check 😓

> Indeed, my original thought was just to cut out the OSE parts that I didn't want or need, but I thought that seemed a little rude, to take someone else's work and then do 20 minutes of work hacking a tiny thing, and slap my name next to theirs like I did something significant!

As long as the GPL license is applied, the proper respect is being shown to the copyright owners (most of whom are not me, the current maintainer). As far as name slapping goes, it's really only important so far as telling everyone who to pester in Discord DMs about bugs and user questions. I certainly don't want those!

**Comment by anthonyronda** [comment]
Closing now, but feel free to ask more questions in our Discord server. A few other maintainers of forks of OSE are chatting with us there. https://discord.gg/9KFQgRkT

I saw you figured out how to make a new release, but keep in mind that the best formulation for your version number does not contain anything except `0-9` and `.` because Foundry may get confused by any other characters in the version field.
